### ISSUE TITLE
ciftify_statclust_report's csv report is inaccurate  when other resolution surfaces file served 

### ISSUE THREAD

**Issue by DearBraveJohn** [edickie/ciftify#116]
I run the ciftify_statclust_report command with a self-downsampled surface(10k per hemisphere) and the overlap information with the atlases in the csv report looks inaccurate, is this situation about the mismatch between surface option file with the default atlases in ciftify?

**Comment by edickie** [comment]
Yes, sorry about that, the atlases it's drawing from are in 32k per hemisphere resolution, and there's no internal resampling step.. I'll add extra checks and warnings into the command.

**Comment by DearBraveJohn** [comment]
Thanks answering, would you consider to add a extra option that users can choose the specific atlas to compute the overlap information.

**Comment by edickie** [comment]
I always intended to add that..(I tried to build the code with the option
of adding it) but I don't know if I figured out what that interface would
look like for the user. Any suggestions?
